src/main/java/lotto/domain/lotto/LottoMoney.java

```diff
@@ -0,0 +1,31 @@
+package lotto.domain.lotto;
+
+import lotto.domain.message.ErrorMessage;
+
+public class LottoMoney {
+
+    public static final int PERCENT = 100;
+
+    private static final int ZERO = 0;
+
+    private final int money;
+
+    public LottoMoney(int money) {
+        validate(money);
+        this.money = money;
+    }
+
+    private void validate(int money) {
+        if (money == ZERO || money % LottoRule.LOTTO_PRICE.getValue() != ZERO) {
+            throw new IllegalArgumentException(ErrorMessage.PRICE_EXCEPTION_MESSAGE.getMessage());
+        }
+    }
```

Comment:
1. validate를 각각 경우에 따라 메소드를 나눠주면 좋을 것 같아요!
2. 가독성을 위해 검증의 조건도 boolean 메소드로 묶어주면 좋을 것 같아요!

```suggestion
    private void validate(int money) {
        validateZero(money);
        validateUnit(money);
    }

    private static void validateUnit(int money) {
        if (isInvalidUnit(money)) {
            throw new IllegalArgumentException(ErrorMessage.PRICE_EXCEPTION_MESSAGE.getMessage());
        }
    }

    private void validateZero(int money) {
        if (isZero(money)) {
            throw new IllegalArgumentException(ErrorMessage.PRICE_EXCEPTION_MESSAGE.getMessage());
        }
    }

    private static boolean isInvalidUnit(int money) {
        return money % LottoRule.LOTTO_PRICE.getValue() != ZERO;
    }

    private static boolean isZero(int money) {
        return money == ZERO;
    }
```

Reply:
예시 코드와 함께 조언 정말 감사합니다. 새로운 시각을 배울 수 있었네요